lib/Models/Terria.ts

```diff
@@ -375,7 +385,9 @@ export default class Terria {
     languageConfiguration: undefined,
     customRequestSchedulerLimits: undefined,
     persistViewerMode: true,
-    openAddData: false
+    openAddData: false,
+    feedbackPreamble: "feedback.feedbackPreamble",
+    feedbackMinLength: 0
```

Comment:
i'm not sure if this needs to be a configuration option as this would allow users to submit a feedback form with no feedback?

Reply:
Till now, it worked without any limitation (user can send completely empty feedback with no info), so I left it like that by default